src/app.ts

```diff
@@ -35,7 +35,15 @@ app.get(
 app.use('/api', apiApp);
 
 async function initDatabase(): Promise<void> {
-  if (!getConnection() && process.env.NODE_ENV !== 'test') {
+  let hasConnection = false;
+  
+  try {
+    getConnection();
+    hasConnection = true;
+  } catch(err) {
+  }
```

Comment:
```suggestion
      // Ignore this; we'll just create a connection now
    }
```

Reply:
Reverting back to a const wouldn't fix it unfortunately. It's the same issue we had with all the mocking before. I'll add the comment though to explain the purpose of it.